### PR TITLE
Fixes #121 Added Ask.com support

### LINKED ISSUE #121: Adding ask.com support
This will be an enhancement to the existing project

### PR DESCRIPTION
**Pull request by sayak119**
Fixes #121 , #165 
- [x] Ask.com feature added
- [x] Icon added
- [x] python 3 compatibility
- [x] spaces and new lines taken care of

### CODE REVIEW

**Comment by sayak119** [comment]
@Pk13055 @nikhilrayaprolu @mariobehling please review.

File: app/scrapers/__init__.py [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
 from yahoo import Yahoo
+from ask import Ask
+
 scrapers = {
-        'g':Google(),
-        'b':Bing(),
-        'y':Yahoo(),
-        'd':Duckduckgo(),
-    }
+
```
Comment (vaibhavsingh97) [inline app/scrapers/__init__.py]: Remove space

**Comment by vaibhavsingh97** [comment]
@sayak119 Please squash commits into one after removing space. If you don't know how to squash just DM me on gitter. I will help 😄

**Comment by sayak119** [comment]
@mariobehling  please merge this feature.

File: app/scrapers/ask.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        return response
+
+    @classmethod
+    def results_search(cls,query):
+        """ Search ask for the query and return set of urls
+        Returns: urls (list)
+                [[Tile1,url1], [Title2, url2],..]
+        """
+        urls=[]
+        response = get_ask_page(query)
```
Comment (cclauss) [inline app/scrapers/ask.py]: See #193

**Pull request merged**